src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -13,4 +13,17 @@ public class CliSyntax {
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_GRADE = new Prefix("g/");
 
+    /* Prefix definitions for edits */
+    public static final Prefix OPTION_PRINT_NAME = new Prefix("!n");
```

Comment:
How come theres a need for different prefixes? Is it for the command box? I think for the command box, maybe you can just check if edit is in the command text then we don't have to add the extra prefixes for edit. Might be a mafan for the user to rmb the prefixes specifically for edit D: